### ISSUE TITLE
CCS1 broke in recent update

### ISSUE THREAD

**Issue by kshedstrom** [NOAA-GFDL/MOM6#436]
```
 h<0 at k=           1 h_old=   9.9999999999999995E-007 wup=   0.0000000000000000      wdn=   3.5324496820976492E-005 dw_dz=  -3.5324496820976492E-005 h_new=  -3.4324496820976495E-005 h_err=   7.8436139407892496E-021

FATAL from PE     0: MOM_regridding: adjust_interface_motion() - implied h<0 is larger than roundoff!


FATAL from PE     0: MOM_regridding: adjust_interface_motion() - implied h<0 is larger than roundoff!

```
It's happening at the corner point just outside to the SW of the domain (4,4) in filtered_grid_motion. This point was part of the OBC segment extensions, but should no longer be included that way, yet the point is still "ocean" in terms of mask2dT and bathyT. Will investigate...

**Comment by MJHarrison-GFDL** [comment]
You can get past this point by removing extensions and comment the corner depths 

https://github.com/ESMG/MOM6/blob/dev/master/src/core/MOM_open_boundary.F90#L1026

**Comment by kshedstrom** [comment]
Yes, I know. I'm working on a segment-ish fix.

On Tue, Mar 14, 2017 at 11:57 AM, Matthew Harrison <notifications@github.com

**Comment by kshedstrom** [comment]
Resolved by d3a8f10
